python/pdo/test/request.py

```diff
@@ -19,6 +19,7 @@
 import time
 import argparse
 import random
+from string import Template
```

Comment:
this is not used?

Reply:
i was using it for one incarnation... will remove later if thats OK